pkg/core/blockchain.go

```diff
@@ -110,6 +110,8 @@ type Blockchain struct {
 	// Current persisted block count.
 	persistedHeight uint32
 
+	p uint32
```

Comment:
p?

Reply:
That's the current state sync point, we need to keep it until state sync is completed. I added a comment.